policy/release/test/test_test.rego

```diff
@@ -0,0 +1,592 @@
+package release.test_test
```

Comment:
Duplicate of `policy/release/test_test.rego` or other way around?

Reply:
I had this issue with other files. I think vscode was saving them or something. Either way I removed this one.